### DISCUSSION TITLE
Suggestion: Commander Pasta

### DISCUSSION THREAD

**Discussion by R2-dT** [blish-hud/Blish-HUD#667] [Module Ideas]
A module which lets people store their copypasta and paste into chat with a click/hotkey. Allow them to organise the different event pastas into folders with quick access during the event.

Some commanders already use macros for their pastas so hopefully a module will bring in more newbie commanders

**Comment by Taschenbuch** [comment]
in addition to adding your own messages, this module could already have a list of all kind of events, meta events, fractals, dungeons, raids, strikes and more. And when you lead one of those you can use the predefined messages to explaint the event/boss/whatever to the squad. instead of directly including those in the module, this could be community based. like a json that you share with others which already have those copy&paste texts inside and you just add this json to a folder where this copypaste module can read it. (like a marker pack for pathing module)

like this tool:
https://old.reddit.com/r/Guildwars2/comments/dogxxi/new_overlayapp_gw2clipboard/

has already been suggested a few times in the suggestion channel, so there is definitely a user base for it.

![grafik](https://user-images.githubusercontent.com/43114787/163731435-5a96d5ec-b06d-4379-aa90-76be9b094591.png)

**Comment by Taschenbuch** [comment]
IS IMPLEMENTED:
chat shorts module
